Update de.m3u

Carlinhos027 opened the pull request:
Deleted duplicate stream, corrected IDs, replaced links & added new channels

Closes #12613 
Closes #12614 
Closes #12615 
Closes #12616 
Closes #12617

Carlinhos027 committed:
Update de.m3u

Deleted duplicate stream, corrected IDs, replaced links & added new channels

Closes #12613 
Closes #12614 
Closes #12615 
Closes #12616 
Closes #12617

BellezaEmporium commented on streams/de.m3u:
```diff
@@ -140,11 +138,11 @@ https://ub1doy938d.gjirafa.net/live/oEZi1otQqIHglY0tzZidyCPWhxrS5WgO/yky0y1.m3u8
 https://api.alpaca.t62a.com/hls/9103/index.m3u8
-#EXTINF:-1 tvg-id="KIKA.de",KiKA [Geo-blocked]
+#EXTINF:-1 tvg-id="KIKA.de",KiKA (1080p) [Geo-blocked]
 https://kikageohls.akamaized.net/hls/live/2022693/livetvkika_de/master.m3u8
```
There's also a worldwide version of Kika, heavily censored for copyright reasons but exists : https://kikahls.akamaized.net/hls/live/2022690/livetvkika_ww/master.m3u8

Carlinhos027 replied:
Added. Thx bro

BellezaEmporium commented on streams/de.m3u:
```diff
@@ -470,21 +468,47 @@ https://edge14.vedge.infomaniak.com/livecast/ik:wir24tv_hms/manifest.m3u8
+#EXTINF:-1 tvg-id="AristoTV.de",Aristo TV (360p)
+https://ma.anixa.tv/clips/stream/aristo/index.m3u8
+#EXTINF:-1 tvg-id="AnixePlus.de",Anixe + (360p)
+https://ma.anixa.tv/clips/stream/anixesd/index.m3u8
```
https://ma.anixa.tv/clips/stream/anixehd/index.m3u8 is Eazzzy TV, if you want to add it.

Carlinhos027 replied:
Is already added.

Carlinhos027 committed:
Update 

Deleted & added Isar TV as alt link to Niederbayern TV Landshut cuz they are the same channel.
Added alt link for Kika & more
